bakery/checkers/namespace.go

```diff
@@ -77,18 +149,43 @@ func (ns *Namespace) ResolveCaveat(cav Caveat) Caveat {
 // ConditionWithPrefix returns the given string prefixed by the
 // given prefix. If the prefix is non-empty, a colon
 // is used to separate them.
-func ConditionWithPrefix(prefix, s string) string {
+func ConditionWithPrefix(prefix, condition string) string {
 	if prefix == "" {
-		return s
+		return condition
 	}
-	return prefix + ":" + s
+	return prefix + ":" + condition
 }
 
 // Register registers the given URI and associates it
 // with the given prefix. If the URI has already been registered,
 // this is a no-op.
 func (ns *Namespace) Register(uri, prefix string) {
```

Comment:
Yes, it's important that it's exported - if you're defining some schema for first party caveats, you'll need to register the schema with the namespace.